2.evaluate_data/identify_sc_outlier_per_plate/identify_per_plate_anomalies.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Get the root directory
+git_root=$(git rev-parse --show-toplevel)
+
+py_path="nbconverted"
+
+# convert all notebooks to python files into the nbconverted folder
+jupyter nbconvert --to python --output-dir="${py_path}/" *.ipynb
+
+# Get the normalized data path (with multiple plates)
+feature_selected_data_path="${git_root}/big_drive/feature_selected_sc_data"
+
+# Check if the folder exists
+if [ -d "$feature_selected_data_path" ]; then
+
+    # Iterate through all Parquet files in the normalized data path
+    for file in "$feature_selected_data_path"/*.parquet; do
+
+        start_time=$(date +%s.%N)
+
+        # Check if each path is a file
+        if [ -f "$file" ]; then
+
+            # Get the plate name from the filename
+            plate_name="$(ls $file | awk -F "/" '{print $NF}' | awk -F "_" '{print $1}')"
+
+            echo ""
```

Comment:
If you want to print a blank line then consider adding a comment stating so. Alternatively, check to ensure that an echo statement is not missing here.

Reply:
I'll just use the -e flag